delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/exporter/ExportEventQuery.scala

```diff
@@ -0,0 +1,19 @@
+package ch.epfl.bluebrain.nexus.delta.sourcing.exporter
+
+import cats.data.NonEmptyList
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{Label, ProjectRef}
+import ch.epfl.bluebrain.nexus.delta.sourcing.offset.Offset
+import io.circe.Decoder
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto.deriveConfiguredDecoder
+
+import scala.annotation.nowarn
+
+final case class ExportEventQuery(id: Label, projects: NonEmptyList[ProjectRef], offset: Offset)
```

Comment:
Is `id` anything other than the folder containing export files? Might be clearer to name it that